src/Libraries/RevitNodes/Elements/Views/Sheet.cs

```diff
@@ -506,6 +521,110 @@ public static Sheet ByNameNumberTitleBlock(string sheetName, string sheetNumber,
             return new Sheet(sheetName, sheetNumber, titleBlockFamilyType.InternalFamilySymbol);
         }
 
+        /// <summary>
+        /// Duplicates A Sheet. 
+        /// </summary>
+        /// <param name="sheet">The Sheet to be Duplicated</param>
+        /// <param name="viewDuplicateOption">Enter View Duplicate Option: 0 = Duplicate. 1 = AsDependent. 2 = WithDetailing.</param>
+        /// <param name="prefix"></param>
+        /// <param name="suffix"></param>
+        /// <returns></returns>
+        public static Sheet DuplicateSheet(Sheet sheet, int viewDuplicateOption = 0, string prefix = "", string suffix = "")
+        {            
+            if (sheet == null)
```

Comment:
The `prefix `and `suffix` is both for view and sheet? I think we should have them seperate for view and sheet.

Reply:
Do you mean that this node should have 2 prefix and suffix?